src/Microsoft.AspNetCore.OData/Results/CreatedODataResult.cs

```diff
@@ -23,22 +23,23 @@ namespace Microsoft.AspNet.OData.Results
     /// <remarks>This action result handles content negotiation and the HTTP prefer header. It generates a location
     /// header containing the edit link of the created entity and, if response has status code: NoContent, also
     /// generates an OData-EntityId header.</remarks>
-    public class CreatedODataResult<T> : IActionResult
+    public class CreatedODataResult<T> : ObjectResult
     {
         private readonly T _innerResult;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="CreatedODataResult{T}"/> class.
         /// </summary>
         /// <param name="entity">The created entity.</param>
-        public CreatedODataResult(T entity)
+        public CreatedODataResult(object entity)
```

Comment:
Why change the parameter type from `T` to `object`?

Reply:
I reverted this change